File: Maximum Value  in a Sequence/max_in_sequence.py
def max_in_sequence(sequence):
    large = sequence[0]

    for i in range(1, len(sequence)):
        if sequence[i] > large:
            large = sequence[i]

    return large

# example sequence:
sequence = [7, 12, 9, 11, 3]
print(len(sequence))

print("The maximum value in the sequence is:", max_in_sequence(sequence))   # prints 12

File: Maximum Value  in a Sequence/max_in_sequence.cpp
#include <iostream>
#include <vector>

using namespace std;

int max_in_sequence(vector<int> sequence) {
    int large = sequence.front();

    for (int i = 1; i < sequence.size(); ++i) {
        if (sequence[i] > large) {
            large = sequence[i];
        }
    }

    return large;
}

int main(void) {
    // example sequence:
    vector<int> sequence = {7, 12, 9, 11, 3};

    // this will print 12
    cout << "The maximum value in the sequence is: " << max_in_sequence(sequence) << endl;

    return 0;
}